DFC.ServiceTaxonomy.ApiFunction/CypherQueries/GetAllOccupationsForSkill.json

```diff
@@ -0,0 +1,8 @@
+﻿{
+	"query": "MATCH (o:esco__Occupation)-[r:esco__relatedEssentialSkill|:esco__relatedOptionalSkill]->(s:esco__Skill {uri:{skill}}), (s)-[:esco__skillType]->(skilltype), (s)-[:esco__skillReuseLevel]->(reuse) WITH{ uri:s.uri, skill:head(s.skos__prefLabel), skilltype:head(skilltype.skos__prefLabel), skillReusability:head(reuse.skos__prefLabel), alternativeLabels:s.skos__altLabel, lastModified:s.dct__modified, occupation:collect({ uri:o.uri, occupation:head(o.skos__prefLabel), relationshipWithSkill:case type(r) when 'esco__relatedEssentialSkill' then 'essential' when 'esco__relatedOptionalSkill' then 'optional' end, alternativeLabels:o.skos__altLabel, lastModified:head(o.dct__modified) }) } as skills return skills",
```

Comment:
This query needs to map some values (e.g. using a case statement, as per relationshipWithSkill), as it returns them as stored in the graph, rather than as specified in the example response.

So skill reuse has to be mapped thus:
"cross-sector skills and competences" -> "Cross-sectoral"
"sector specific skills and competences" -> "Sector-specific"
"occupation specific skills and competences" -> "Occupation-specific"
"transversal skills and competences" -> "Transversal"

_* note: I'd check the required capitalisation with @developerbuzz, as all other data values are shown in all lower case._

Skill type also needs to be mapped:
"skill" -> "competency"
"knowledge" -> "knowledge"

Reply:
We should fix the same in the other queries, such as GetAllSkillsForOccupation.